component/authentication-endpoint/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -15,6 +15,8 @@
 # under the License.
 
 certificateNotFound.error.message=Could not find X509 certificate in browser!
-unknown.error.code=500
+userNotFound.error.message=Couldn't find the username for X509Certificate's attribute!
+userNamesConflict.error.message=Couldn't find any certificate belongs to this user!
+unknown.error.code=400
```

Comment:
Is this HTTP 400 error code?

Reply:
Not to specific to HTTP 400, we set this bad request error code when is there any bad request perform when validating with x509 certificate